team/requirements.txt

```diff
@@ -1,6 +1,48 @@
-Flask==0.10.1
```

Comment:
We went over this, Karan.  We're using the new updated version of flask

Reply:
O.K.

Reply:
This is the new Flask version - Flask==0.12.2, so it should be backward compatible.